.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "dlib"]
+	path = dlib
+	url = https://github.com/davisking/dlib.git
```

Comment:
Please transfer these repositories to stream-labs if possible.

Reply:
Don't have a permission to create a stream-labs repository
If access is given or created repositories Ic an change links